corehq/apps/settings/forms.py

```diff
@@ -292,7 +292,8 @@ def __init__(self, *args, user_domains=None, max_allowed_expiration_days=None, t
 
         user_domains = user_domains or []
         all_domains = (self.ALL_DOMAINS, _('All Projects'))
-        self.fields['domain'].choices = [all_domains] + [(d, d) for d in user_domains]
+        default_empty_choice = ('', '')
```

Comment:
Should we be more explicit that this option is not going to work out?
```suggestion
        default_empty_choice = ('', _('(None)'))
```